src/components/Header/Header.test.tsx

```diff
@@ -1,12 +1,29 @@
+import * as React from "react";
+import Cookies from "js-cookie";
 import { render, screen, waitFor, within } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 import { axe } from "jest-axe";
-import * as React from "react";
 import renderer from "react-test-renderer";
 
 import Header from "./Header";
 import { refineryResponse } from "./components/SitewideAlerts.test";
 
+const patronResponse = {
+  data: {
+    patron: {
+      barCodes: ["01234567890123"],
+      emails: ["test@nypl.org"],
+      id: "0123456",
+      names: ["PATRON, JANE"],
```

Comment:
Okay, so it seems there is no issue if there's a middle initial, right?

Reply:
Thought about it some more and seems like no!